src/Tizen.NUI.Components/Controls/Navigation/Navigator.cs

```diff
@@ -24,8 +24,33 @@
 namespace Tizen.NUI.Components
 {
     /// <summary>
-    /// The Navigator is a class which navigates pages with stack methods such
-    /// as Push and Pop.
+    /// TransitionFinishedEventArgs is a class to record page TransitionFinished event arguments which will be sent to user.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class TransitionFinishedEventArgs : EventArgs
+    {
+    }
+
+    /// <summary>
+    /// The Navigator is a class which navigates pages with stack methods such as Push and Pop.
+    ///
+    /// With Transition class, Navigator supports smooth transition of View pair between two Pages
+    /// by using PushWithTransition(Page) and PopWithTransition() methods.
+    /// If there is View pair of current top Page and next top Page those have same View.TransitionOptions.TransitionTag,
+    /// Navigator creates smooth transition motion for them.
+    /// Navigator.Transition property can be used to set properties of the Transition such as TimePeriod and AlphaFunction.
+    /// When all transitions are finished, Navigator calls a callback methods those connected on the "TransitionFinished" event.
+    /// 
+    /// example
```

Comment:
There are `<example>` and `<code>` xml tags. How about using them?

Reply:
Thank you